t/variation.t

```diff
@@ -105,6 +105,47 @@ my $expected_result = { rs142276873 => $expected_variation_1, rs67521280 => $exp
 
 is_json_POST($base,$post_data,$expected_result,"Try to POST list of variations");
 
+# Test population allele frequency with no allele_count
+  $id = 'COSM946275';
+  my $expected_pops_nc = {
+    'source' => 'http://cancer.sanger.ac.uk/cancergenome/projects/cosmic/',
+    'mappings' => [
+      {
+        'location' => '13:25744274-25744274',
+        'assembly_name' => 'GRCh37',
+        'end' => 25744274,
+        'seq_region_name' => '13',
+        'strand' => 1,
+        'coord_system' => 'chromosome',
+        'allele_string' => 'C/T',
+        'start' => 25744274
+     }
+    ],
+    'name' => 'COSM946275',
+    'MAF' => undef,
+    'ambiguity' => 'Y',
+    'populations' => [
```

Comment:
Is this susceptible to random order changes?

Reply:
Unless I have misunderstood https://metacpan.org/pod/Test::Deep#HASH-COMPARISONS, it shouldn't be.

Reply:
'populations' is a list generated from the Variation code. Test::Deep isn't allowed to do set operations on a list by default. @helensch Is the major allele always first, or is there some other sorting mechanism?

Reply:
@nerdstrike yes the test it is susceptible to random order change in the 'populations' list.
As Iit is the populations that I want to compare I suggest doing  a cmp_deeply with a bag comparison
cmp_deeply($pops_json->{'populations'}, 
             bag(@{$expected_pops_nc->{'populations'}}), 
          "Returning population information - no allele_count");

Reply:
Sounds reasonable